python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -1060,29 +1069,35 @@ def deblend(
             shape and region of the sky.
         mDeconvolved :
             The deconvolved images of the same shape and region of the sky.
-        catalog :
+        parentCatalog :
             The merged `SourceCatalog` that contains parent footprints
-            to (potentially) deblend. The new deblended sources are
-            appended to this catalog in place.
+            to (potentially) deblend. If a parent is subdivided into
+            multiple deconvolved parents, the deconvolved parents are
+            added to this catalog in place.
+        objectCatalog :
```

Comment:
Out of order (did you actually want objectCatalog to come first?)

Reply:
It doesn't matter, so I switched the docstring order.